src/components/auth/torus/AuthTorus.js

```diff
@@ -143,6 +173,11 @@ const AuthTorus = ({ screenProps, navigation, styles, store }) => {
         // const userExists = await userStorage.userAlreadyExist()
         log.debug('checking userAlreadyExist', { exists, fullName })
         const { source } = await ready(replacing)
+        showLoadingDialog(true)
+        await delay(300)
```

Comment:
why fixed delay for 30ms is used ? what we're waiting for ?

probably better to do
```javascript
showLoadingDialog(true)

const { source } = await ready(replacing)

hideDialog()
```

Reply:
wanted the new dialog with "V" icon to be seen by user. otherwise it will immediatly disappaer
in anycase this has been changed to the lottie animation checkmark we have